src/main/java/xades4j/verification/SignatureSpecificVerificationOptions.java

```diff
@@ -81,6 +87,19 @@ public SignatureSpecificVerificationOptions checkKeyUsage(boolean enabled)
         return this;
     }
 
+    /**
+     * Configures whether to check the validity (time) of the signer certificate
+     * If enabled (the default) signature validation will fail if certificate is invalid.
+     *
+     * @param enabled {@code true} to enable the check, {@code false} to disable
+     * @return the current instance
+     */
+    public SignatureSpecificVerificationOptions checkValidity(boolean enabled)
```

Comment:
The changes on this file can also be reverted, right?

Reply:
I'm not sure why I missed this.